ragen/env/game_2048/env_api.py

```
@@ -0,0 +1,272 @@
+import gymnasium as gym
+import numpy as np
+from typing import Tuple, Any, Dict, List
```

Ruff (0.14.14)

[warning] `typing.Tuple` is deprecated, use `tuple` instead

(UP035)